### ISSUE TITLE
Check Data validation by the API

### ISSUE THREAD

**Issue by giorgioriccardi** [giorgioriccardi/wp-submit-form#5]
Related #2 
> The plugin should use the WordPress REST API to submit the data, and the data should be `validated` on both front end and `by the API endpoint`.

```
// Redirection $success
function ssws_redirect_success() {
	$current_url = $_SERVER['REQUEST_URI'];
	if (strpos($current_url, '?') == true) {
		$url_parameters = $current_url."&ssws-endpoint=success";
	} else {
		if (substr($current_url, -1) == '/') {
			$url_parameters = $current_url."?ssws-endpoint=success";
		} else {
			$url_parameters = $current_url."/?ssws-endpoint=success";
		}
	}
	return $url_parameters;
}
```

Ref: [https://rigor.com/blog/how-to-monitor-and-validate-data-from-api-endpoints](https://rigor.com/blog/how-to-monitor-and-validate-data-from-api-endpoints)

**Comment by giorgioriccardi** [comment]
API Validation started, first response is 500:
https://github.com/giorgioriccardi/wp-submit-form/commit/c141f96babd0c162131d03085a268e2c61aaf1f7

**Comment by giorgioriccardi** [comment]
merged success remote response in version 2.4.0
